### PR TITLE
Update machine-learning-terms.md

### PR DESCRIPTION
**Pull request by creatornadiran**
I added an explanation about the loss functions.

### CODE REVIEW

File: explanations/machine-learning-terms.md [hunk 1]
```diff
@@ -70,6 +70,24 @@ Researchers often invent complicated names for simple things. This page will hel
     </ul>
 </td>
 </tr>
+<tr>
+<td valign="top">Loss (Cost) Functions</td>
```
Comment (haltakov) [inline explanations/machine-learning-terms.md]: I would skip the Cost part here and just call it Loss Function. You can mention later in the text that it is sometimes also called a cost function.

File: explanations/machine-learning-terms.md [hunk 2]
```diff
@@ -70,6 +70,24 @@ Researchers often invent complicated names for simple things. This page will hel
...
+<td>
+    <p>Functions that help to evaluate your model.</p>
```
Comment (haltakov) [inline explanations/machine-learning-terms.md]: I think this needs to be a bit more precise. The loss function is important during training and is usually not very suitable to evaluate the performance of the model.

Maybe something like: "A function that is used to evaluate the performance of a model and guide the training process."

File: explanations/machine-learning-terms.md [hunk 3]
```diff
@@ -70,6 +70,24 @@ Researchers often invent complicated names for simple things. This page will hel
...
+    <p>Loss Functions show the error between what value your model predicts and what the value actually is. Gradient descent is applied to these functions to find the least error. Some common ones are:</p>
```
Comment (haltakov) [inline explanations/machine-learning-terms.md]: Maybe mention that it needs to be differentiable.

**Review by haltakov — CHANGES_REQUESTED** [review]
Thanks for your contribution! I like your description and the examples, but I would like to make some small changes to make it even better. I left some comments with a couple of ideas. I will be glad to hear your opinion on that!

**Comment by creatornadiran** [comment]
Thank you for your ideas. I agree with all your comments and made the necessary changes. Please let me know if I'm still missing something.

**Review by haltakov — APPROVED** [review]
Cool, thanks for the modifications! Let's merge this.
